contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -427,8 +427,9 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
       }
```

Comment:
The names in the comment above don't appear to match the variable names in the code which is causing me to become confused.  Could this PR include an update to that comment starting at line 413?

Reply:
Also, on line 425, can you explain what `iszero` is doing? I am confused as to why we would call `iszero` on the value returned by `gt`.

Also, my understanding is this logic for sending fulfillments was lifted from other products.  Does a similar problem exist in those contracts too?

Reply:
> Also, my understanding is this logic for sending fulfillments was lifted from other products. Does a similar problem exist in those contracts too?

`gasUsed` is the only difference from CCIP, can't speak to where other products are at

Reply:
@KuphJr I believe that's a short-hand for "less than or equal to" which [doesn't have its own op-code](https://ethervm.io/).